src/js/data-store.ts

```diff
@@ -230,6 +231,16 @@ class DataStore {
     this.firebaseImp.saveUserData(this.presenceMap[uuid]);
   }
 
+  predictionFor(basestationId: string):Prediction {
+    if (this.predictions[basestationId] !== undefined) {
+      return this.predictions[basestationId];
+    }
+    return {
+      temp: -1,
+      rationale: "(no answer)"
+    } // TBD: What do do when there is no prediction?
```

Comment:
I would think that `null` would be a better response here and clients have to deal with the possibility that there might not be a prediction. Showing nothing in the chip would make more sense than showing `-1` (not that we'd ever show the -1, but it would be easier to mistake it for a valid prediction than a null response).